src/main/resources/JDK-sef-methods.txt

```diff
@@ -658,18 +978,62 @@ java.util.Date.equals(java.lang.Object)
 java.util.Date.hashCode()
 java.util.Date.toString()
 java.util.Deque.contains(java.lang.Object)
-java.util.Deque.isEmpty()
```

Comment:
This change is unexpected. I'll find out.

Reply:
Turns out this is an abstract function with no method body - I think this means that we can ignore these as anything that subclasses them will have their isEmpty marked as pure alongside their implementation.

Reply:
Added context - @Pure is in included in the checker-framework source files.